.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: '
+labels: ''
+assignees: ''
+
+---
+
+## Bug Report
+
+### Describe the bug
+
+A clear and concise description of what the bug is.
+
+### To Reproduce
+
+Detailed steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected behavior
+
+A clear and concise description of what you expected to happen.
+
+### Screenshots
+
+If applicable, add screenshots to help explain your problem.
+
+### Desktop (please complete the following information):
+
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
+
+### Smartphone (please complete the following information):
```

Comment:
I think this session for backend does not apply, maybe you could add SO instead.

Reply:
Ok, I will remove for Smartphone.